absorber_ds/absorber.py

```diff
@@ -0,0 +1,150 @@
+"""
+This module contains Absorber Device Class and run method for it
+"""
+
+# Imports
+from facadedevice import Facade, proxy_attribute, logical_attribute, triplet
+from tango import DevState, AttrWriteType
+from tango.server import command
+
+
+class Absorber(Facade):
+    """
+    Tango facade device class for vacuum absorbers
+
+    This class contains four water flow alarms:
+
+    * FSW_1
+    * FSW_2
+    * FSW_3
+    * Achromat_FSW
+
+    as well as attributes to hold names of PLCs for those alarms.
+
+    There is also a whole mechanism of maintaining absorber:
+
+    * name of PLCs for inserting and extracting absorberx
+    * name of PLCs for changing state of absorber (inserted or extracted)
+    * attribute for holding value of absorber state (inserted or extracted)
+    * methods to Insert or Extract the absorber itself
+    """
+    # device initialization
+
+    def safe_init_device(self):
+        """
+        This is safe_init_device method overriden from Facade base class. It's
+        used to initialize facade device safely
+        """
+        super(Absorber, self).safe_init_device()
+        self.set_state(DevState.ON)
+        self.set_status("Device is running")
+
+    # proxy attributes
+
+    waterFlowAlarm_Achromat = proxy_attribute(
+        dtype=bool,
+        access=AttrWriteType.READ,
+        property_name="PlcAttrName_AchromatFSW",
+        description="PLC device and attribute name for alarm from water flow "
+                    "in the achromat")
+
+    PlcAttrName_Extract = proxy_attribute(
```

Comment:
If you use CamelCase for naming, abbreviations (such as PLC) should be all-capitals. Therefore, this attribute should be named: `PLCAttrName_Extract`. Please fix accordingly all other attributes' and properties' as well.